ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/base-journey/journey-base.ts

```diff
@@ -1,10 +1,11 @@
-import { EventEmitter } from '@angular/core';
-import { JourneyStep } from './journey-step';
+import {EventEmitter} from '@angular/core';
+import {JourneyStep} from './journey-step';
 
 export abstract class JourneyBase {
     readonly redirect: EventEmitter<JourneyStep>;
+    isSubmitted: boolean;
```

Comment:
Despite existing for both journey I would rather see this as private in the sub-classes, it feels like nothing outside of the journey should need to care about whether it's submitted or not.